package.json

```diff
@@ -1,117 +1,104 @@
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint --silent"
-    }
-  },
-  "eslintConfig": {
-    "extends": "@10up/eslint-config",
-    "env": {
-      "es6": true,
-      "node": true
-    },
-    "rules": {
-      "array-bracket-spacing": [
-        "error",
-        "always"
-      ],
-      "indent": [
-        "error",
-        4,
-        {
-          "SwitchCase": 1
-        }
-      ],
-      "no-constant-condition": [
-        "error",
-        {
-          "checkLoops": false
-        }
-      ],
-      "no-console": "off",
-      "no-empty": [
-        "error",
-        {
-          "allowEmptyCatch": true
-        }
-      ],
-      "object-curly-spacing": [
-        "error",
-        "always"
-      ],
-      "prefer-destructuring": [
-        "error",
-        {
-          "array": false,
-          "object": true
-        }
-      ],
-      "require-jsdoc": "off",
-      "yoda": [
-        "error",
-        "never"
-      ]
-    }
-  }
+	"name": "wp-local-docker",
+	"version": "2.8.0-rc.15",
+	"description": "WP Local Docker",
+	"license": "MIT",
+	"homepage": "https://10up.github.io/wp-local-docker-docs/",
+	"bugs": {
+		"url": "https://github.com/10up/wp-local-docker-v2/issues"
+	},
+	"keywords": [
+		"wordpress",
+		"wp",
+		"wp-cli",
+		"wpsnapshots",
+		"docker",
+		"docker-compose",
+		"nginx",
+		"php",
+		"mysql",
+		"phpmyadmin",
+		"memcache",
+		"memcached",
+		"mailcatcher"
+	],
+	"preferGlobal": true,
+	"main": "index.js",
+	"files": [
+		"src",
+		"global",
+		"scripts",
+		"index.js",
+		"hosts.js"
+	],
+	"scripts": {
+		"postinstall": "npm run 10updocker postinstall --silent",
+		"10updocker": "./index.js",
+		"10updocker-hosts": "./hosts.js",
+		"test": "jest",
+		"lint": "eslint . --ext .js --ignore-pattern node_modules",
+		"format": "npm run lint --silent -- --fix"
+	},
+	"bin": {
+		"10updocker": "./index.js",
+		"10updocker-hosts": "./hosts.js"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git@github.com:10up/wp-local-docker-v2.git"
+	},
+	"engines": {
+		"node": ">=12"
+	},
+	"author": "10up <info@10up.com> (https://10up.com/)",
+	"contributors": [
+		{
+			"name": "Chris Marslender"
+		}
+	],
+	"dependencies": {
+		"@sindresorhus/slugify": "^1.1.0",
+		"asyncro": "^3.0.0",
+		"boxen": "^4.2.0",
+		"chalk": "^4.1.0",
+		"docker-compose": "^0.23.4",
+		"dockerode": "^3.2.1",
+		"fs-extra": "^9.0.1",
+		"hostile": "^1.3.2",
+		"inquirer": "^7.3.3",
+		"log-symbols": "^4.0.0",
+		"markdown-it": "^11.0.1",
+		"mkcert": "^1.4.0",
+		"mkcert-prebuilt": "^1.0.0",
+		"mysql": "^2.17.1",
+		"netcat": "^1.4.0",
+		"nodegit": "^0.27.0",
+		"ora": "^5.1.0",
+		"read-yaml": "^1.1.0",
+		"shell-escape": "^0.2.0",
+		"strip-ansi": "^6.0.0",
+		"sudo-prompt": "^9.2.1",
+		"table": "^6.0.3",
+		"terminal-link": "^2.1.1",
+		"update-check": "^1.5.3",
+		"which": "^2.0.2",
+		"write-yaml": "^1.0.0",
+		"yargs": "^16.0.3"
+	},
+	"devDependencies": {
+		"@10up/eslint-config": "^1.0.9",
```

Comment:
It would be great to update this to 2.x at some point.

Reply:
@eugene-manuilov @nicholasio any reason not to do that now?  No reason this would affect the 2.8 release as far as I could image.

Reply:
the rules changed quite a bit, there might be things that can't be fixed automatically. I'll defer to @eugene-manuilov

Reply:
@eugene-manuilov seems like holding off on bumping `@10up/eslint-config` to 2.x makes most sense so as to not further delay the 2.8.0 release, yeah?

Reply:
Yeah, it's not critical for the 2.8.0 release and can be done later.